src/pages/index.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import { Tag, Transaction } from '@components/tags';
+import { Flex, Stack } from '@blockstack/ui';
 
-const Home: React.FC = () => <>Home</>;
+// @ts-ignore
+const Home: React.FC = () => (
+  <Flex height="100vh" width="100%" align="center" justify="center">
+    <Stack spacing="loose">
+      {Object.keys(Transaction).map(label => (
+        <Tag
+          type={
+            // @ts-ignore
```

Comment:
`<Tag type={Transaction[label as keyof typeof Transaction]} />`

// @ts-ignore is big boo-boo in TypeScript land

Reply:
yeah but this is just to display them :) this won't happen in real life

Reply:
will update though!